src/toM3u8.js

```diff
@@ -15,6 +15,14 @@ export const formatAudioPlaylist = ({ attributes, segments }) => {
 };
 
 export const formatVttPlaylist = ({ attributes, segments }) => {
+  if (segments === undefined) {
```

Comment:
`undefined` referenced directly. We should check our linter, but `typeof segments === 'undefined'` should work